docs/manual/scala/guide/services/code/ServiceClients.scala

```diff
@@ -74,7 +74,7 @@ package circuitbreakers {
           .withCircuitBreaker(CircuitBreaker.identifiedBy("hello2"))
       )
     }
-    //#circuit-breakers
+    //#circuit-breaker
     // @formatter:on
```

Comment:
The validation is running, I think the problem is that the validator is based on older logic than what the renderer uses - the validator looks for segments by doing a simple `String.contains` on each line, and this line does contain `#circuit-breaker`, so it passed.  But later on logic was added to the renderer to do a more intelligent match, which allowed segments to be prefixes of each other, which wasn't allowed before.  But the validator was never updated for this.  This all happened years ago in Play, so the same problem probably exists in the Play docs.